### ISSUE TITLE
Maintain zoom level between tabs and pages

### ISSUE THREAD

**Issue by atla5** [scantailor/scantailor#244]
"The interface allows one to zoom in, but does not maintain the zoom level between pages. 

It is frustrating to have to zoom every page."

_via BookBuff: https://groups.google.com/forum/#!topic/scantailor-users/1T5Dam4kVp0_

The zoom level resets even between tabs on the output page, making dewarping more of a hassle than it needs to be.

**Comment by ospalh** [comment]
On second thought, i agree that it would be nice to keep the zoom level when switching from *tab* to *tab*. I still think going back to 100 % when switching from *step* to *step*, like from Margins to Output is OK.

**Comment by atla5** [comment]
Yeah, tab to tab is really what's been bugging me because it does get in the way of dewarping.
